strum_macros/src/macros/strings/from_string.rs

```diff
@@ -136,7 +136,7 @@ fn try_from_str(
         #[allow(clippy::use_self)]
         impl #impl_generics ::std::convert::TryFrom<&str> for #name #ty_generics #where_clause {
             type Error = #strum_module_path::ParseError;
-            fn try_from(s: &str) -> ::std::result::Result< #name #ty_generics , Self::Error> {
+            fn try_from(s: &str) -> ::std::result::Result< #name #ty_generics , <Self as ::std::convert::TryFrom<&str>>::Error> {
```

Comment:
Can you switch this to `::core::convert`? #189 was supposed to fix this and then #192 fixed it again.

Reply:
Fixed. Thanks for supporting no_std.